SmartDeviceLink/private/SDLControlFramePayloadRPCStartServiceAck.m

```diff
@@ -23,9 +24,12 @@ @interface SDLControlFramePayloadRPCStartServiceAck ()
 @property (copy, nonatomic, readwrite, nullable) NSArray<NSString *> *secondaryTransports;
 @property (copy, nonatomic, readwrite, nullable) NSArray<NSNumber *> *audioServiceTransports;
 @property (copy, nonatomic, readwrite, nullable) NSArray<NSNumber *> *videoServiceTransports;
+@property (strong, nonatomic, readwrite, nullable) SDLVehicleType *vehicleType;
 
 @end
 
+SDLVehicleType *sdl_parseVehicleType(BsonObject *const payloadObject);
+BOOL sdl_putStringValue(BsonObject *const payloadObject, const char *key, NSString *value);
```

Comment:
this looks like a private function definition; if its a must they should be inside the interface declaration